tests/infer_data_path.py

```diff
@@ -367,6 +264,17 @@ def get_infer_test_data():
     test_data["STARCODER1-15B-base"]["max_output_len"] = 128
     test_data["STARCODER1-15B-base"]["max_batch_size"] = 5
 
+    test_data["STARCODER2-15B-base"] = {}
+    test_data["STARCODER2-15B-base"]["model_type"] = "starcoder"
+    test_data["STARCODER2-15B-base"]["min_tps"] = 1
+    test_data["STARCODER2-15B-base"]["location"] = "Local"
+    test_data["STARCODER2-15B-base"]["model_dir"] = "/tmp/STARCODER2-15B-base/trt_llm_model-1/"
+    test_data["STARCODER2-15B-base"]["checkpoint"] = "/opt/checkpoints/starcoder-2_15b_4k_vfinal/4194b.nemo"
```

Comment:
Naming convention is abused here. Could we make it consistent with other models from `/opt/checkpoints`?

Reply:
There are these two models available, maybe we can use them here:
```
STARCODER2-15B-base
STARCODER2-15B-lc
```